benchmarks/templates/benchmarks/profile-table.html

```diff
@@ -0,0 +1,60 @@
+{% load static %}
+
+{% if models %}
+<form>
```

Comment:
would it be possible to use the same table template as for the main page? I'm worried we'll change that at some point and will run into discrepancies.
(If the difference arises from the checkbox, we could just not display it for the main overview table)